src/AppBundle/Subscriber/AutoLabelPRFromContentSubscriber.php

```diff
@@ -109,7 +109,7 @@ private function getValidLabels()
             'Asset', 'BC Break', 'BrowserKit', 'Bug', 'Cache', 'ClassLoader',
             'Config', 'Console', 'Critical', 'CssSelector', 'Debug', 'DebugBundle',
             'DependencyInjection', 'Deprecation', 'Doctrine', 'DoctrineBridge',
-            'DomCrawler', 'Drupal related', 'DX', 'Easy Pick', 'Enhancement',
+            'DomCrawler', 'DotEnv', 'Drupal related', 'DX', 'Easy Pick', 'Enhancement',
```

Comment:
@chalasr Actually components and labels exist so Carson should be aware of them even if they are only in master to be able to do its job ;)